exe/roo_on_rails

```diff
@@ -36,6 +36,11 @@ module RooOnRails
       end
 
       def run
+        # Force output to be unbuffered, otherwise the CI
+        # tests will fail because the parent app won't see
+        # the log output from the test app
+        $stdout.sync = true
```

Comment:
Do you need both this and the line in the run_test_app?

Reply:
Yes, I think so! When it tests sidekiq, it starts the `exe/roo_on_rails` entrypoint. For other tests, it calls `bin/rails`.